example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -81,6 +77,7 @@ async function setupCollectionAndTestData(config, testSuffix, testDataFolder) {
     '@cumulus/test-data/granules/BROWSE.MYD13Q1.A2002185.h00v09.006.2015149071135.1.jpg',
   ];
 
+  await removeCollectionAndAllDependencies({ config, collection });
```

Comment:
One of the more vexing failure situations for this test occurs when the static collection used for CMR export fails cleanup - the tests then were failing continually *and* failing on cleanup resulting in a required stack manual intervention for this test to ever pass. 

Short of randomizing this collection, making the cleanup more robust helps, but up-front ensuring the stack state with respect to the MYD13Q1 collection ensures that if bamboo/other factors result in a bad cleanup state the test will start with a relatively fresh slate.    99% of the time this should functionally do nothing.